packages/teleport/src/components/BannerList/Banner.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Box, Flex, Text } from 'design';
+import { Cross, Info, Warning } from 'design/Icon';
+
+type Props = {
+  message: string;
+  // The backend defines the severity as an integer value with the current
+  // pre-defined values: LOW: 0; MEDIUM: 5; HIGH: 10
+  severity: number;
+  id: string;
+  onClose: (id: string) => void;
+};
+
+export function Banner({ id, message = '', severity = 0, onClose }: Props) {
+  return (
+    <Box bg={getBackgroundColor(severity)} p={3}>
+      <Flex alignItems="center">
+        {generateIcon(severity)}
+        <Text bold>{message}</Text>
```

Comment:
lets center this text as we do for our expired license and switchback banner

Reply:
Left aligned text (in English) is superior for legibility and imho is more pleasing to the eye in this situation.

Reply:
i still think it should be centered and fixed to the top, otherwise i'll clash, or we make the other banners left aligned (the switchback banner might look weird)
![image](https://user-images.githubusercontent.com/43280172/187762081-15ebff5d-65e4-4fc7-be41-51f20da48bfc.png)

Reply:
So the license one will be moving to this new banner system but the switchback one will remain the same as the messaging is long vs short form